man/crypttab.xml

```diff
@@ -138,6 +133,15 @@
         size is then given by the key size.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>keyfile-erase=</option></term>
+
+        <listitem><para>Takes a boolean argument. If enabled the specified keyfile is erased after the volume
+        is activated, or when activation fails. This is in particular useful when the key file is only
```

Comment:
"If enabled, the specified keyfile is erased after the volume is activated or when activation fails."